Resources/Prototypes/_Harmony/Objectives/base_objectives.yml

```diff
@@ -0,0 +1,15 @@
+﻿# teach lesson
+- type: entity
+  abstract: true
+  parent: BaseTargetObjective
+  id: BaseTeachLessonObjective
+  components:
+  - type: Objective
+    unique: false
+    icon:
+      sprite: Objects/Weapons/Melee/fireaxe.rsi
+      state: icon
+  - type: ObjectiveBlacklistRequirement
+    blacklist:
+      components:
+      - SocialObjective
```

Comment:
This prototype should be declared in `traitor.yml` with `TeachLessonRandomPersonObjective` instead